nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/BigMappedMatrixFile.java

```diff
@@ -275,7 +273,7 @@ public double sumValues() throws IOException {
                 sum += getMapping(mapN).getFloat(offN);
             } catch (IOException e) {
                 MappedByteBuffer mapping = getMapping(mapN);
-                System.out.println(mapN + " Err " + offN + " " + mapping.capacity() + " " + mapping.limit());
+                log.error("{} Err {} {} {}", mapN, offN , mapping.capacity(), mapping.limit());
                 System.exit(0);
```

Comment:
Changed to log.error since immediately followed by System.exit....the exit code is 0 though, but this isn't a normal exit